agent/src/softwarecontaineragent_dbus_stub.cpp

```diff
@@ -0,0 +1,322 @@
+#include "softwarecontaineragent_dbus_stub.h"
+
+Glib::ustring interfaceXml0 = R"xml(
```

Comment:
Does this really work? Using "xml" as delimiter should stop processing on the first line I think. Even if it happens to work, choose something that does not show up in the xml code, like "endOfXMLCode".